backend/reg.ml

```diff
@@ -299,6 +299,15 @@ let set_of_array v =
            if i >= n then Set.empty else Set.add v.(i) (add_all(i+1))
          in add_all 0
 
+let set_has_collisions s =
+  let phys_regs = Hashtbl.create (Set.cardinal s) in
+  Set.fold (fun r acc ->
+    match r.loc with
+    | Reg id ->
+      if Hashtbl.mem phys_regs id then true
+      else (Hashtbl.add phys_regs id (); acc)
+    | _ -> acc) s false
```

Comment:
(Again, does not matter that much for
such a function, but please avoid
catch-all patterns.)

Reply:
Fixed